docs/software/switch-config.md

```diff
@@ -87,12 +127,18 @@ Ardupilot Modes don't line up very well with the standard -100% (988us) to +100%
 
 ### Why do you keep saying "*put arm on AUX1*"?
 
-For safety reasons.
+For safety and performance reasons.
+
+**SAFETY**
 
 AUX1 is sent with every packet going out, this is the most reliable way to be able to tell your model to disarm. If your arm switch is in another aux channel, it may be several packets before that switch is transmitted, and there's no guarantee that the RX will actually receive that packet. There's a non-trivial chance your model **may not ever disarm** if the link quality is low and it just so happens that the packet containing the arm switch is getting missed every time. Forcing the arm switch into every packet on AUX1 means that if **any** packet is received by ExpresLRS, it will disarm your model, not just a less than 1-in-7 chance.
 
 It also protects against unintentional disarms caused by a corrupt packet changing the value of the arm switch to disarmed. Betaflight requires that 4x "disarm" commands are received before disarming to guard against this possibility. With arm on AUX1, a single corrupt packet can not disarm your model. With arm on AUX2-8, the one corrupt switch value will be sent 6 times before the value is refreshed, but the flight controller would have already disarmed by that point.
 
+**PERFORMANCE**
+
+Your transmitter and receiver act differently when “armed” and when “disarmed”. When disarmed, the transmitter and receiver increase telemetry in order to talk back and forth with each other faster. This makes configuring the devices more responsive. When disarmed, everything will appear to be working appropriately including the channels. However, when disarmed, the packet frequency is low and telemetry ratios are high and many safe guards such as locking out the configuration interface are not yet enabled. When you arm, the transmitter and receiver are fully enabled with safeguards and the packet and telemetry rates that you selected and the system starts to monitor signal strength, packet loss, and adjusting transmit power, etc. Arming is an important part of the performance of the control link. Please use Channel 5 (Aux 1) as indicated.

```

Comment:
All of this paragraph is inaccurate. TLM Boost (1:2 mode) can happen even while armed and is dependent on MSP traffic, not IsArmed.  

`IsArmed` Controls
* Dynamic Power will not raise power in all cases when disarmed
* All "Button" inputs are disabled if armed
* All "Joystick" (5-way buttons) are disabled if armed
* Bump to Share disabled if armed
* VTX Admin disabled if armed
* Integrated VTX channel change disabled if armed
* Race telemetry mode relies on Armed status
* Something about integrated thermal-based fan control is disabled when armed? Not clear on this one

Reply:
Awesome. Is this the right location for this list? For example, the RFMD mode index is on the Signal Health Page. So under my table I put a link to that index instead of recreating the list. Should this "disabled on arm" list go on this doc or some other doc? @CapnBry

Reply:
I think this list can go onto this doc. I did remove the link to RF Modes because I did not feel those needed to be distinctly explained in the scope of the document (you can re-add that If you would like, I just would like a bit more tie in)

Reply:
I'm not sure there's any better place, I put it here when the docs were in the wiki and there wasn't much talk about it anywhere else. I think here is good and if there's a more relevant page found somewhere else later, we can move the whole section.

Reply:
Perfect, I'll worth sleuth it and update.